math.gobra

```diff
@@ -0,0 +1,62 @@
+// Package math provides common math functions for verification
+package math
+
+// Max returns the maximum of x and y
+ghost
+pure func Max(x, y int) int {
+    return x > y ? x : y
```

Comment:
Please make sure that these are all tabs and not spaces, as this is what is standard in Go